src/spv/bitcoin/BRWallet.cpp

```diff
@@ -829,7 +960,15 @@ BRTransaction *BRWalletCreateTxForOutputs(BRWallet *wallet, const BRTxOutput out
     minAmount = BRWalletMinOutputAmountWithFeePerKb(wallet, MIN_FEE_PER_KB);
 
     wallet->lock.lock();
-    feeAmount = _txFee(wallet->feePerKb, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
+    if (feeRate > 1000)
+    {
+        feeAmount = _txFee(feeRate, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
+    }
+    else
+    {
+        feeAmount = _txFee(wallet->feePerKb, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
+    }
+
```

Comment:
We can do
```
feeRate = std::max(feeRate, wallet->feePerKb);
feeAmount = _txFee(feeRate, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
```

Reply:
Changed in https://github.com/DeFiCh/ain/pull/308/commits/a6aae235b22b244bd371a191bb58f0d12c8faab7

Added checking that fee is not less than 1 Sat/byte.